host/generate-composite.sh

```diff
@@ -5,6 +5,21 @@
 DIR=$(dirname $0)
 IFS=$'\n'
 
+select_version() {
+    ## Selects the directory of the version to be generated
+    ## Defaults to v3 for now 
+    if [ "$VERSION" == "4" ]; then
+        echo "Version 4 selected"
+        DIR=$(dirname $0)/4/bullseye/amd64/base
+    elif [ "$VERSION" == "3" ]; then
+        echo "Version 4 selected"
+        DIR=$(dirname $0)/3.0/buster/amd64/base
+    else
+        echo "No valid version selected. Defaulting to v3"
```

Comment:
thinking it might be better to not use defaults and throw if the argument is not set. avoids having to update the default at a later time and things potentially breaking silently.
since we control the caller side, we can make sure they  always pass explicit version.

Reply:
Makes sense. I will remove the default version so that it throws. And run a quick test

Reply:
Resolving this comment as I was able to implement and test. Thanks Bala